index.html

```diff
@@ -239,11 +239,11 @@ <h3>
           run the <dfn>steps to determine the visibility state</dfn>:
         </p>
         <ol class="algorithm">
-          <li>Let |doc:Document| be the {{Document}} of the <a>top-level
-          browsing context</a>.
-          </li>
-          <li>If the {{Document/defaultView}} of |doc| is `null`, return
-          {{VisibilityState["hidden"]}}.
+          <li>Return "hidden" if the <a>context object</a>is not [=Document/fully active=]</li>
+          <li>Let |doc:Document| be the <a>active document</a> of the top level
```

Comment:
It's "top-level browsing context". We should link it to https://html.spec.whatwg.org/#top-level-browsing-context also.